src/BMW_E65.cpp

```diff
@@ -15,7 +15,7 @@ uint8_t A90=0xe9;//0x0A9 first counter byte
 uint8_t A91=0x00;//0x0A9 second counter byte
 uint8_t BA5=0x4d;//0x0BA first counter byte(byte 5)
 uint8_t BA6=0x80;//0x0BA second counter byte(byte 6)
-
+uint8_t AA1=0x0F;
```

Comment:
```suggestion
uint8_t AA1=0x0F;//0x0AA first counter byte(byte 1)
```

Since all the other counters have a comment to describe what they are, it probably makes sense to do that for the new counter as well.